UM/Qt/qml/UM/CategoryButton.qml

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2022 Ultimaker B.V.
+// Uranium is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.2
+import QtQuick.Controls 2.1
+
+import UM 1.5 as UM
+
+Button
+{
+    id: base
+
+    height: enabled ? UM.Theme.getSize("section_header").height : 0
+
+    property var expanded: false
+
+    property alias arrow: categoryArrow
+    property alias categoryIcon: icon.source
+    property alias labelText: categoryLabel.text
+
+    property color text_color:
+    {
+        if (!base.enabled)
```

Comment:
Shouldn't this also be handled in the states?

Reply:
https://github.com/Ultimaker/Uranium/pull/794/commits/a0faefcab0bc85706975cd7d429cded2e30cfc63